src/Controllers/NucleiTest.php

```diff
@@ -129,14 +129,6 @@ public function run(Request $request): Response
             $acsUrl = (string)$acsUrl;
         }
 
-        $target = parse_url($acsUrl, PHP_URL_HOST);
```

Comment:
What's the reason of not using host as target? Was something wrong with it? What if we implement a test which is not SAML related, would the target still be ACS URL?

Reply:
It is just an optimization, the behavior is the same as the original. My thinking was that
- we don't have to specify both host and ACS URL (the host was generated from ACS URL anyway)
- nuclei will extract the host from the ACS URL (instead of the original PHP code, the host is not needed on the PHP side)
- If we run any more tests, like HTTPS checks, they (still) will be executed against the ACS hostname, which I think makes sense, since that is the hostname we want to check for problems

Reply:
Makes total sense...